runtime/runtime/src/lib.rs

```diff
@@ -1120,11 +1164,19 @@ impl Runtime {
             });
         }
 
+        let gas_used_for_migrations =
+            if apply_state.migration_context.is_first_block_with_current_version {
```

Comment:
Is it possible that this code will not be reached in the desired block because of this condition above?

`if !apply_state.is_new_chunk
            && apply_state.current_protocol_version
                >= ProtocolFeature::FixApplyChunks.protocol_version()`

AFAIU it means that the block doesn't have a presented chunk, which may be the case.

Reply:
Thanks, fixed